src/strings/normalize.c

```diff
@@ -561,9 +561,20 @@ static MVMint32 should_break(MVMThreadContext *tc, MVMCodepoint a, MVMCodepoint
                 return 0;
             if ( GCB_b == MVM_UNICODE_PVALUE_GCB_ZWJ )
                 return 0;
+            if ( GCB_b == MVM_UNICODE_PVALUE_GCB_GLUE_AFTER_ZWJ )
+                return 0;
+            if ( b == 0x2640 || b == 0x2642 )
```

Comment:
Maybe we can `#define` constants for these rather than magic numbers in the code?